test/expectations/expectations_test_runner.rb

```diff
@@ -42,13 +42,13 @@ def default_args
 
         required_ruby_version = ruby_requirement_magic_comment_version(path)
         if required_ruby_version && RUBY_VERSION < required_ruby_version
-          class_eval(<<~RB)
+          class_eval(<<~RB, __FILE__, __LINE__ + 1)
```

Comment:
This is so cool and I had no idea you could do that!

Reply:
Is there documentation anywhere that talks about this or do you just know about it from experience?

Reply:
Many blog posts mentioned it but it looks like the [source code comment doesn't provide any example](https://rubyapi.org/3.1/o/module#method-i-class_eval). It feels like an opportunity to contribute!